init-tools.cmd

```diff
@@ -57,29 +46,12 @@ if NOT exist "%DOTNET_LOCAL_PATH%" (
   exit /b 1
 )
 
-:afterdotnetinstall
-
-if exist "%DOTNET_SDK_CMD%" goto :afterdotnetsdkinstall
-
-echo Installing dotnet sdk...
-set /p DOTNET_SDK_VERSION=< "%~dp0DotnetCLISdkVersion.txt"
-if NOT exist "%DOTNET_SDK_PATH%" mkdir "%DOTNET_SDK_PATH%"
-set DOTNET_ZIP_NAME=dotnet-dev-win-x64.%DOTNET_SDK_VERSION%.zip
-set DOTNET_REMOTE_PATH=https://dotnetcli.azureedge.net/dotnet/Sdk/%DOTNET_SDK_VERSION%/%DOTNET_ZIP_NAME%
-set DOTNET_LOCAL_PATH=%DOTNET_SDK_PATH%%DOTNET_ZIP_NAME%
-echo Installing '%DOTNET_REMOTE_PATH%' to '%DOTNET_LOCAL_PATH%' >> "%INIT_TOOLS_LOG%"
-powershell -NoProfile -ExecutionPolicy unrestricted -Command "$retryCount = 0; $success = $false; do { try { (New-Object Net.WebClient).DownloadFile('%DOTNET_REMOTE_PATH%', '%DOTNET_LOCAL_PATH%'); $success = $true; } catch { if ($retryCount -ge 6) { throw; } else { $retryCount++; Start-Sleep -Seconds (5 * $retryCount); } } } while ($success -eq $false); Add-Type -Assembly 'System.IO.Compression.FileSystem' -ErrorVariable AddTypeErrors; if ($AddTypeErrors.Count -eq 0) { [System.IO.Compression.ZipFile]::ExtractToDirectory('%DOTNET_LOCAL_PATH%', '%DOTNET_SDK_PATH%') } else { (New-Object -com shell.application).namespace('%DOTNET_SDK_PATH%').CopyHere((new-object -com shell.application).namespace('%DOTNET_LOCAL_PATH%').Items(),16) }" >> "%INIT_TOOLS_LOG%"
-if NOT exist "%DOTNET_LOCAL_PATH%" (
-  echo ERROR: Could not install dotnet cli correctly. See '%INIT_TOOLS_LOG%' for more details. 1>&2
-  exit /b 1
-)
-
-:afterdotnetsdkinstall
+:afterdotnetrestore
```

Comment:
This name doesn't really make sense to me.  We aren't calling "dotnet restore" before this.  I think it should be left as it is: `afterdotnetinstall`.